helper/fetch-release

```diff
@@ -28,7 +28,7 @@ curl --silent -L -O $baseUrl/nar-hash.txt.asc
 
 # Verify signature for nar-hash
 gpg --homedir $GPG_HOME --verify nar-hash.txt.asc &> /dev/null || {
-    echo "Error: Signature verification failed. Please open an issue in the project repository."
+    >&2 echo "Error: Signature verification failed. Please open an issue in the project repository."
```

Comment:
Running `fetch-release > nix-bitcoin-release.nix` still wipes the `nix-bitcoin-release.nix`

Reply:
I've now completely removed `fetch-release > nix-bitcoin-release.nix` from the docs.